README.md

```diff
@@ -62,3 +61,26 @@ for Maven, add the following sections to your pom.xml (replacing $LATEST_VERSION
     </dependency>
   </dependencies>
 ```
+
+
+### S3 Blockstore
+
+By default, Nabu will construct a File based blockstore
+
+To use S3, replace existing /blocks mount and configure:
```

Comment:
I don't think we should support migrating to S3 which is what this sounds like. They can just spin up a new instance. It needs to be configurable for first run.